src/app/page.tsx

```diff
@@ -1,113 +1,56 @@
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Learn about Next.js in an interactive course with&nbsp;quizzes!
-          </p>
-        </a>
+function surahCard() {
+  return (
+    <div className={classCardDiv}>
+      <p className={classCardTitle}>Surah 188 Al-Mumtahanah</p>
+      <p className={classCardData}>Murojaah by using Memory</p>
+      <p className={classCardData}>Total Murojaah is 100</p>
+      <p className={classCardDate}>Sat, Feb 10 &apos;24</p>
+    </div>
+  );
+}
 
-        <a
-          href="https://vercel.com/templates?framework=next.js&utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Templates{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Explore starter templates for Next.js.
-          </p>
-        </a>
+function ayahCard() {
+  return (
+    <div className={classCardDiv}>
+      <p className={classCardTitle}>Ayah 200 to 208</p>
+      <p className={classCardData}>Surah 1 Al-Baqarah</p>
+      <p className={classCardData}>Murojaah by using Memory</p>
+      <p className={classCardData}>Total Murojaah is 100</p>
+      <p className={classCardDate}>Sat, Feb 10 &apos;24</p>
+    </div>
+  );
+}
 
-        <a
-          href="https://vercel.com/new?utm_source=create-next-app&utm_medium=appdir-template&utm_campaign=create-next-app"
-          className="group rounded-lg border border-transparent px-5 py-4 transition-colors hover:border-gray-300 hover:bg-gray-100 hover:dark:border-neutral-700 hover:dark:bg-neutral-800/30"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          <h2 className={`mb-3 text-2xl font-semibold`}>
-            Deploy{" "}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50 text-balance`}>
-            Instantly deploy your Next.js site to a shareable URL with Vercel.
-          </p>
-        </a>
-      </div>
-    </main>
+export default function Home() {
+  return (
+    <div className="flex flex-col mt-[72px] pt-4 px-4">
+      {juzCard()}
+      {surahCard()}
+      {ayahCard()}
+      {repeat([juzCard(), surahCard(), ayahCard()], 7)}
```

Comment:
is this for dummy purposes?

Reply:
yes, later will change to retrieve data from api